packages/ts/react-crud/src/media-query.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react';
+
+export function useMediaQuery(query: string): boolean {
+  const [matches, setMatches] = useState(() => window.matchMedia(query).matches);
```

Comment:
What about using [`useMedia`](https://github.com/streamich/react-use/blob/master/docs/useMedia.md) from `react-use` here? That would reduce the amount of code we have to support.

Reply:
I'd prefer not to add a new dependency for a single hook. The argument is not really sound, as a dependency is something you have to maintain.

Reply:
Ok, let's have this implementation for now